tracer/src/Datadog.Trace/Util/UnmanagedMemoryPool.cs

```diff
@@ -0,0 +1,148 @@
+﻿// <copyright file="UnmanagedMemoryPool.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
```

Comment:
should it be #nullable enable? as well as the next one?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4302/commits/67e36ce21878425b551c2b0b5b08631e9452b028